server/conf/evolutions/default/76.sql

```diff
@@ -2,7 +2,7 @@
 
 ALTER TABLE applications ADD COLUMN IF NOT EXISTS latest_note varchar;
 
--- Update historical data from application events table
+# -- Update historical data from application events table
```

Comment:
Isn't this where the error is coming from? Is the problem maybe that this is now doing two inline comments? (Since both # and -- are ways to specify a single line comment?) I would think it's valid, but maybe SQL is weird.

Reply:
the 44.sql has comments like this all over the place but not sure why this one is complaning. I will reset it locally and give it another shot.